sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3094,6 +3032,12 @@ object SQLConf {
     .timeConf(TimeUnit.MILLISECONDS)
     .createWithDefault(5000)
 
+  val STREAMING_REAL_TIME_MODE_ALLOWLIST_CHECK = buildConf(
+    "spark.sql.streaming.realTimeMode.allowlistCheck")
```

Comment:
nit. Indentation looks a little wrong to me for all realTimeMode configurations, @jerrypeng and @viirya .
- spark.sql.streaming.realTimeMode.minBatchDuration
- spark.sql.streaming.realTimeMode.allowlistCheck

It would be great if you can match with the other code around here.